pkgs/development/python-modules/dateparser/0.x.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, mock
+, parameterized
+, pytestCheckHook
+, dateutil
+, pytz
+, regex
+, tzlocal
+, convertdate
+, umalqurra
+, jdatetime
+, ruamel_yaml
+}:
+
+buildPythonPackage rec {
+  pname = "dateparser";
+  version = "0.7.6";
+
+  src = fetchFromGitHub {
+    owner = "scrapinghub";
+    repo = "dateparser";
+    rev = "v${version}";
+    sha256 = "0j3sm4hlx7z0ci5fnjq5n9i02vvlfz0wxa889ydryfknjhy5apqw";
+  };
+
+  checkInputs = [
+    mock
+    parameterized
+    pytestCheckHook
+  ];
+
+  pytestFlagsArray = [ "tests" ];
+
+  disabledTestPaths = [
+    "tests/test_dateparser_data_integrity.py" # ImportError: No module named ruamel.yaml
```

Comment:
@jonringer Can you explain this? Maybe it's related to https://github.com/NixOS/nixpkgs/commit/a4fc8c4b938f7ec88d6f2ecee3338bf8b5c351bc.

Reply:
probably a PEP420 related issue, where only one of the ruamel packages can be imported.

Reply:
or I avoided having to add another dependency. I don't remember

Reply:
So we should probably add `pythonNamespace` to all ruamel packages?

Reply:
I think this predated it.

It's totally plausible I wanted to avoid a dependency on raumel.yaml